src/components/badge/badge.stories.mdx

```diff
@@ -73,4 +73,4 @@ If the counter is less than 1 badge will not show itself
 
 ### Badge
 
-<ArgsTable of={Badge} />
+<ArgsTable of={Badge} />
```

Comment:
Not sure what was updated here or why it needed to be?

Reply:
I think it was accidental commit from Designer's Notes Removal ....